src/app/CommandSender.h

```diff
@@ -308,6 +310,8 @@ class CommandSender final : public Messaging::ExchangeDelegate
      */
     CHIP_ERROR SetCommandSenderConfig(ConfigParameters & aConfigParams);
 
+    void SetExchangeManager(Messaging::ExchangeManager * apExchangeMgr);
+    void SetCommandSenderDelegate(Callback * apCallback);
```

Comment:
I guess these are being added to allow creating a CommandSender without this info and then set it later?

But why do we want to allow that?  I don't think we do.

Reply:
We create the CommandSender object as a member of the ICDRefreshKeyInfo but we don't have these information at that point. Especially, the Exchange Mgr is available only after the CASE session is established.

Reply:
Updated with Optional.